### PR TITLE
Update bibliography

### PR DESCRIPTION
**Pull request by kba**

### CODE REVIEW

File: site/de/publications.md [hunk 1]
```diff
@@ -8,103 +8,113 @@ lang-ref: publications
 ## Publikationen
```
Comment (EEngl52) [inline site/de/publications.md:8]: those are the presentations, not the publications. publications should stay on top of the document though

File: site/en/publications.md [hunk 2]
```diff
@@ -8,103 +8,113 @@ lang-ref: publications
 ## Publications
```
Comment (EEngl52) [inline site/en/publications.md:8]: publications and presentations confused as in the german version

File: site/de/publications.md (hunk 1 above)
Comment (kba) [reply site/de/publications.md:8]: fixed in 3afafc85

File: site/en/publications.md (hunk 2 above)
Comment (kba) [reply site/en/publications.md:8]: fixed in 3afafc85

**Pull request merged**